### ISSUE TITLE
VBA template: adopt V1 and V3 components where components are not used

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17095]
## Description
The VBA needs the following updates to components:

[Cleveland VA Regional Benefit Office](https://web-txcrk8bym4a0ivw3xfr8xvdhezmls36f.demo.cms.va.gov/cleveland-va-regional-benefit-office/locations/cleveland-va-regional-benefit-office/) (This is a tugboat)

### Elements that are not using components that should be

**Updated 3/20**

~Alert - Expandable
I believe the Operating status alert should work like Vet Center not VAMC? Right now it's like VAMC and linking to a 404 Operating Status page. I think it should be a [Alert-expandable component](https://design.va.gov/components/alert-expandable) and not link to a separate page.~

1. ~Telephone~
   - ~VA benefits hotline number should be updated to use the `message-aria-describedby` prop: https://design.va.gov/components/telephone#aria-described-by~
   - ~Both phone numbers (Standard and TTY number) in the "Can't find services" block are updated to use the component~
2. Spotlights
   - Per the comment from [Amanda below ](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17095#issuecomment-2009844648), we will be updating the spotlights to use [the card component](https://design.va.gov/components/card#default---white-with-border). 
   - [Figma design with full annotation notes.](https://www.figma.com/file/ikCp2eOubIgbN1TwyEYIrN/VAMC-Master?type=design&node-id=428%3A3013&mode=design&t=aRQ2J73hbbiuFEsR-1)

  
**Future future state:** 
![Screenshot 2024-03-22 at 12 04 32 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/137226108/6f08d218-9110-433a-ae95-a6913fc51ab7)

## Engineering notes
We will adapt spotlights on other products to use the card component for spotlights as well. Anything we do here could be reusable, if we're thoughtful about implementation being reusable / extensible.

## Acceptance Criteria
- [ ] Spotlights on use card component. Cases to handle:
    - [ ] National and local spotlights
    - [ ] any standard link in the CTA uses action link class
    - [ ] any email in the CTA is an email link (no component available)
    - [ ] any phone number in the CTA, if possible, use the telephone component. If Drupal updates block this / make it hard, add a comment and treat it as a standard link.
    - [ ] without CTA
    - [ ] Any link or phone number in the description will not use a component
- [x] Alert-expandable component has been added
- [ ] Update and run existing test(s).
- [ ] If additional tests are needed, stub a ticket to note this for future work.
- [ ] Requires accessibility review

**Comment by laflannery** [comment]
@xiongjaneg I did a component review of VBA when I was reviewing Max's PR this morning. There are some updates we can do if we can fit them in with the current work I just don't know what epic to put this in

**Comment by xiongjaneg** [comment]
@laflannery How about #14933 for now?

**Comment by laflannery** [comment]
@xiongjaneg I added the phone number update to #14957 because that made sense and that work is being done now. The other 2 (breadcrumbs and operating status seemed out of scope?) so I wonder if this ticket should just be either pulled in separately or we can work those in if/as they come up?

**Comment by xiongjaneg** [comment]
@laflannery I'll stub a ticket for those separately.

**Comment by maxx1128** [comment]
I would estimate this ticket at a **3** since the larger number of components to update increases the scope and number of elements to check.

**Comment by mmiddaugh** [comment]
Is there anything remaining to be done in this issue @maxx1128 @eselkin ?
- `On this page` has no V3 version and is in the Icebox
- `Telephone` was added to #14957 to be completed as part of that ticket
- `Alert - Expandable` was demoed by Max today

**Comment by aklausmeier** [comment]
We need to include any elements that should be using components but are not as it will become a [staging blocker](https://dsva.slack.com/archives/C03R5SBELQM/p1710857760673329). 

These should use the cards component. @thejordanwood is updating design to match design system card component. 

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/137226108/2d96dda5-ba6b-4e96-88ba-927f4590eac4)

**Comment by laflannery** [comment]
@mmiddaugh I reviewed the current state of VBA and confirmed that there were still a couple instances of the phone number that needed some tweaks. I have updated the ticket description with those specific instances as well as the new spotlight updates from Amanda

**Comment by aklausmeier** [comment]
updated ticket with a link to the finalized annotated Figma design

**Comment by laflannery** [comment]
Telephone moved to #16148

**Comment by eselkin** [comment]
Moving to cards is a 3. Moving to action links for some types of links and statements of type of email or phone links makes it a 5. Moving just to action links is a 3 still if not differentiating link types.

**Comment by jilladams** [comment]
Final answer: 3. Because: right now we're moving spotlights to a card component, and all CTAs will regular links. 

In future future state: we will do Spotlight Drupal work on fields, and in FE will then split links into some action links, some not.

**Comment by jilladams** [comment]
@eselkin Laura found that the Service Locations also are not using the card component on VBA templates. That won't block SL launch, but: noting that it's now included in the ticket here to update anything possible on VBA to use components. Please update estimate, if it changes this from a 3.

**Comment by laflannery** [comment]
Note in reference to @jilladams's [comment](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17095#issuecomment-2078136355) that we included Service locations to update to the Card component:

I have now removed this because with PW help Josh got to this sooner that we anticipated and mucking around with Service Locations a few days before we planned to launch seemed like abad planned. So I pulled that out into [it's own ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18097). [For more context/decision see slack thread](https://dsva.slack.com/archives/C0FQSS30V/p1715276396817459?thread_ts=1715266902.041569&cid=C0FQSS30V)

**Comment by davidmpickett** [comment]
@eselkin In order to Verify this on SL Tugboat, we'll need to rebase https://github.com/department-of-veterans-affairs/content-build/pull/1912 and then rerelease content yes?

**Comment by laflannery** [comment]
We don't need to have this on the SL integration branch - we can just preview on staging or prod

**Comment by laflannery** [comment]
I previewed in Prod and there isn't any space between the spotlights when there are multiple
![Screenshot 2024-05-10 at 4 38 47 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/5f777e16-bc34-47e5-91ac-c50b73100a07)

If it helps determine the spacing/classes needed, we have this on Vet Centers already with the proper spacing for when there are 1, 2, or 3 spotlights present:
- [Example with 3](https://www.va.gov/boston-vet-center/)
- [Example with 2](https://www.va.gov/brooklyn-vet-center/)

**Comment by laflannery** [comment]
Confirmed on Prod preview:
![Screenshot 2024-05-16 at 9 24 48 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/c11513f7-84be-4ba5-9d75-d5b384653b3b)
